comixed-web/src/app/library/services/comic-list.service.spec.ts

```diff
@@ -32,7 +32,7 @@ import { Frame, Subscription } from 'webstomp-client';
 import {
   COMIC_LIST_UPDATE_TOPIC,
   LOAD_SCAN_TYPES_MESSAGE,
-  SCAN_TYPE_ADD_QUEUE
+  SCAN_TYPE_UPDATE_TOPIC
```

Comment:
This imports are not used here

Reply:
It's being used in ScanTypeService as part of the subscription to scan type updates:

```
        this.logger.trace('Subscribing to scan type updates');
        this.subscription = this.webSocketService.subscribe<ScanType>(
          SCAN_TYPE_UPDATE_TOPIC,
          scanType => {
            this.logger.debug('Received scan type:', scanType);
            this.store.dispatch(scanTypeAdded({ scanType }));
          }
        );
        this.webSocketService.send(LOAD_SCAN_TYPES_MESSAGE, '');
      }
```

Reply:
Removed the unneeded imports.